buildbot/tox_setup.sh

```diff
@@ -1,74 +1,102 @@
 #!/usr/bin/env bash
-# pyenv root
-export PYENV_ROOT="$HOME/.pyenv"
-TEST_ROOT=$PWD/..
 
-# Install pyenv if it's missing
-if [[ ! -d $PYENV_ROOT ]]; then
-    git clone git://github.com/yyuu/pyenv.git ${PYENV_ROOT}
-    cd ${PYENV_ROOT}
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+if [[ ! -d $PYENV_ROOT ]]
```

Comment:
I intended that to both check for existence of the variable *and* the directory to which it points, in one shot.